target/linux/ramips/dts/mt7628an_zbtlink_zbt-we2426-b.dts

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "zbtlink,zbt-we2426-b", "mediatek,mt7628an-soc";
+	model = "Zbtlink ZBT-WE2426-B";
+
+	aliases {
+		led-boot = &led_wlan;
+		led-failsafe = &led_wlan;
+		led-upgrade = &led_wlan;
+		label-mac-device = &wmac;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 38 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		lan1 {
+			label = "green:lan1";
```

Comment:
`label`property is deprecated, please convert it to `color`/`function` properties. Then, an additional header file needs to be included.

example:

```
#include <dt-dindings/leds/common.h>

leds {
    compatible = "gpio-leds";

    led-0 {
        gpios = <&gpio 43 GPIO_ACTIVE_LOW>;
        color = <LED_COLOR_ID_GREEN>;
        function = LED_FUNCTION_LAN;
        function-enumerator = <1>;
    };
};
```

Reply:
Labels converted to color/function.
Thanks for your help.

Reply:
LED IDs changed "led-0" .. "led-5".